index.html

```diff
@@ -1,534 +1,1024 @@
+            </p>
+          </div>
+        </div>
+        <div class="features__feature features__feature-m">
+          <div class="features__icon features__icon-m">
+            <svg>
+              <use xlink:href="img/icons.svg#icon-credit-card"></use>
+            </svg>
+          </div>
+          <div class="features__deatail-cont">
+            <h5 class="features__header">Dreptul Transporturilor</h5>
+            <p>
+              Asistență în cazuri legate de accidente de vehicule și alte
+              aspecte legate de transport.
+            </p>
+          </div>
+        </div>
+        <div class="features__feature features__feature-m">
+          <div class="features__icon features__icon-m">
+            <svg>
+              <use xlink:href="img/icons.svg#icon-credit-card"></use>
+            </svg>
+          </div>
+          <div class="features__deatail-cont">
+            <h5 class="features__header">Asistare în materie contractuală</h5>
+            <p>
+              Ajutor în înțelegerea și negocierea contractelor pentru a proteja
+              interesele clienților.
+            </p>
+          </div>
+        </div>
+        <div class="features__feature features__feature-m">
+          <div class="features__icon features__icon-m">
+            <svg>
+              <use xlink:href="img/icons.svg#icon-credit-card"></use>
+            </svg>
+          </div>
+          <div class="features__deatail-cont">
+            <h5 class="features__header">
+              Reprezentare în materie contractuală
+            </h5>
+            <p>
+              Ajutor în înțelegerea și negocierea contractelor pentru a proteja
+              interesele clienților.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
 
-  <section class="section" id="presa">
-    <div class="section__title section__title--testimonials">
-      <h2 class="section__description">Vezi ce spun și alții</h2>
-      <h3 class="section__header">
-        Apariții presă
-      </h3>
-    </div>
+    <section class="section section--sign-up">
+      <div class="section__title">
+        <h3 class="section__header">
+          Ai nevoie de asistență juridică personalizată?
+        </h3>
+      </div>
+      <button class="btn">Trimite mesaj</button>
+      <a href="tel:++40742891577"><button class="btn">Sună</button></a>
+    </section>
 
-    <div class="slider">
+    <section class="section" id="presa">
+      <div class="section__title section__title--testimonials">
+        <h2 class="section__description">Vezi ce spun și alții</h2>
+        <h3 class="section__header">Apariții presă</h3>
+      </div>
 
-      <button class="slider__btn slider__btn--left">&larr;</button>
-      <button class="slider__btn slider__btn--right">&rarr;</button>
-      <div class="dots"></div>
-    </div>
-  </section>
+      <div class="slider">
+        <div class="slide" style="transform: translateX(-800%)">
```

Comment:
Please refactor inline styles to a dedicated CSS class. For this and following classes I would create CSS classes like `t-8`, `t-7`, etc and use them to keep the HTML more clean.